examples/js/controls/EditorControls.js

```diff
@@ -37,8 +37,36 @@ THREE.EditorControls = function ( object, domElement ) {
 
 	this.focus = function ( target ) {
 
+		if ( target === undefined || !target.isObject3D ) {
```

Comment:
Hmm, lights and cameras extend `Object3D` so I'm not sure about the last check here...

Reply:
That's the idea, this is only guarding against some random non-3d JavaScript object being passed in. The camera should focus on lights and cameras and groups too.

Reply:
Hmm, what random non-3d JavaScript object? And how could it get there?

Reply:
For example with the code: EditorControls.focus( 7 );
Maybe I'm being too protective... I can take it out if you think its not necessary?

Reply:
Yeah, we don't do type checking in general. I would take it out 👍